themes/bootstrap3/templates/header.phtml

```diff
@@ -1,119 +1,122 @@
-<?php $account = $this->auth()->getManager(); ?>
+<?php
+/** @var \Laminas\View\Renderer\PhpRenderer $this */
+$account = $this->auth()->getManager();
+?>
 <div class="banner container navbar">
-  <div class="navbar-header">
-    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#header-collapse" aria-label="<?=$this->transEscAttr('mobile_toggle_navigation_text')?>">
-      <?=$this->icon('ui-menu') ?>
-    </button>
-    <a class="navbar-brand lang-<?=$this->layout()->userLang ?>" href="<?=$this->url('home')?>">VuFind</a>
-  </div>
-  <?php if (!isset($this->layout()->renderingError)): ?>
-    <div class="collapse navbar-collapse" id="header-collapse">
-      <nav>
-        <ul class="nav navbar-nav navbar-right flip">
-          <?php if ($this->feedback()->tabEnabled()): ?>
-            <li>
-              <a class="icon-link" id="feedbackLink" data-lightbox href="<?=$this->url('feedback-home') ?>">
-                <?=$this->icon('feedback', 'icon-link__icon') ?>
-                <span class="icon-link__label"><?=$this->transEsc('Feedback')?></span>
-              </a>
-            </li>
-          <?php endif; ?>
-          <?php $cart = $this->cart(); ?>
-          <?php if ($cart->isActive()): ?>
-            <li id="cartSummary" aria-live="polite" aria-atomic="true">
-              <a id="cartItems" class="icon-link" data-lightbox title="<?=$this->transEscAttr('View Book Bag')?>" href="<?=$this->url('cart-home')?>">
-                <?=$this->icon('cart', 'icon-link__icon') ?>
-                <span class="icon-link__label">
+    <div class="navbar-header">
```

Comment:
It looks like your IDE has changed our 2-space template indent to a 4-space indent here, creating a lot of diffs that aren't actually necessary.